src/InteractiveWindow/VisualStudio/VsInteractiveWindowCommandFilter.cs

```diff
@@ -269,6 +281,11 @@ private int PreEditorCommandFilterExec(ref Guid pguidCmdGroup, uint nCmdID, uint
             {
                 switch ((VSConstants.VSStd97CmdID)nCmdID)
                 {
+                    // TODO: Add support of rotating clipboard ring 
+                    // https://github.com/dotnet/roslyn/issues/5651
+                    case VSConstants.VSStd97CmdID.PasteNextTBXCBItem:
+                        return VSConstants.E_FAIL;
```

Comment:
Added filter in `PreEditorCommandFilterQueryStatus` as @amcasey suggested. However, based on how `CommandIds.BreakLine` is handled in `PreEditorCommandFilterExec`, it seems we don't really need to do anything for `case VSConstants.VSStd97CmdID.PasteNextTBXCBItem`?

@dpoeschl So what is the right way to handle it in this case?